
Yet another hack for this hacky installation ...

  - Rene Rebe <rene@rocklinux.org>

--- rox-2.1.3/install.sh	2003-11-23 17:16:58.000000000 +0100
+++ rox-2.1.3/install.sh	2004-09-29 16:46:41.265314758 +0200
@@ -203,9 +203,9 @@
 endir "$MIMEDIR"
 endir "$MIMEDIR/packages"
 cp rox.xml "$MIMEINFO" || die "Failed to create $MIMEINFO"
-update-mime-database "$MIMEDIR" || die "update-mime-database failed
-Make sure you have installed the shared MIME database from:
-http://www.freedesktop.org/software/shared-mime-info"
+#update-mime-database "$MIMEDIR" || die "update-mime-database failed
+#Make sure you have installed the shared MIME database from:
+#http://www.freedesktop.org/software/shared-mime-info"
 
 echo "Installing application..."
 endir "$APPDIR"
